tsdb/block.go

```diff
@@ -40,8 +40,7 @@ import (
 // IndexWriter serializes the index for a block of series data.
 // The methods must be called in the order they are specified in.
 type IndexWriter interface {
-	// AddSymbols registers all string symbols that are encountered in series
-	// and other indices. Symbols must be added in sorted order.
+	// AddSymbol register a single symbol.
```

Comment:
Grammar nit: register → registers

I believe `AddSymbol` still has to be called in sorted order. If so, that would be nice to include in the doc comment. Similar to the last sentence that was there before. Something like:

`// AddSymbol registers a single symbol. Symbols must be registered in sorted order.`

Reply:
Indeed, the expected parameters are sorted.It has been revised again. Please review,thx.

Reply:
Thank you for your preciseness.It has been revised again.😁